okhttp/src/main/java/okhttp3/Handshake.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2019 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okhttp3
+
+import okhttp3.internal.Util
+import java.io.IOException
+import java.security.Principal
+import java.security.cert.Certificate
+import java.security.cert.X509Certificate
+import java.util.ArrayList
+import javax.net.ssl.SSLPeerUnverifiedException
+import javax.net.ssl.SSLSession
+
+/**
+ * A record of a TLS handshake. For HTTPS clients, the client is *local* and the remote server
+ * is its *peer*.
+ *
+ * This value object describes a completed handshake. Use [ConnectionSpec] to set policy
+ * for new handshakes.
+ */
+class Handshake private constructor(
+    private val tlsVersion: TlsVersion,
+    private val cipherSuite: CipherSuite,
+    private val peerCertificates: List<Certificate>,
+    private val localCertificates: List<Certificate>
```

Comment:
nit: these should be +2 not +4

Reply:
Done 26a75752e3edd3600fbc2f4c0e796134e3b08277